specification/07-theory_operations.adoc

```diff
@@ -431,4 +431,98 @@ TSM ->> VMM: [COVH] - spdm_covh_connect_device()
 
 === Interface Assignment
 
+Once both the SPDM session and the IDE link are secured and established, the
+host supervisor domain manager may directly assign a TDI to a TVM, through the
+`COVH` interface. This is a four steps process:
+
+1. The host supervisor domain manager initiates the interface assignment flow by
+   having the TSM move the TDI into the TDISP `LOCKED` state.
+2. The TVM verifies and accepts the locked TDI into its TCB.
+3. The host supervisor domain manager asks the TSM to move the TDI to the TDISP
+   `RUNNING` state.
+4. The TVM verifies that the TDI is in the TDISP `RUNNING` state and starts
+   using it.
+
+The next section describe in more details what step 2 covers, i.e. which steps
+a TVM should follow in order to be able to verify a locked TDI and accept or
+reject it into its TCB.
+
+==== TDI Acceptation
+
+It is the TVM responsibility to accept or reject the assigned TDI into its
+TCB, and to explicitly notify both the TSM and the host supervisor domain
+manager about its decision. The TVM should check for the following security
+attributes before being able to decide whether or not it can safely add a TDI
+into its TCB:
+
+1. **SPDM session establishment**: A secured SPDM session must be established
+   between the TDI’s DSM and the TSM. TVM verifies that attribute from the TSM,
+   through the `sbi_covg_get_device_link()` `COVG` ABI.
+2. **IDE link**: The PCIe physical link between the Root Port and the physical
+   device must be confidentiality and integrity protected through IDE. As for
+   the SPDM session, the TVM calls into the `sbi_covg_get_device_link()` `COVG`
+   ABI to verify that attribute from the TSM.
+3. **Firmware update policy**: The TVM must verify that the device firmware
```

Comment:
I think firmware update policy is just part of TDI report.

I suggest to generalize it and change title to `TDI report`.

I suggest to change API to be `sbi_covg_get_tdi_report`, then TVM can check all field in the TDI report, such as INTERFACE_INFO.

Reply:
> I think firmware update policy is just part of TDI report.

Shouldn't we also include the SPDM session termination setting to those attributes. The combination of those 2 will define the device firmware update policy, including if the device can be runtime updated. I think the TVM lust be made aware of at least those 2 settings in order to decide if it accepts it or not. Another attribute that should be added there is the Device SPDM measurement freshness capabilities (`MEAS_FRESH_CAP`) in order for the TVM to check how it may handle runtime updates. So overall, I think it's more than just the TDI interface report, which is why I'm proposing to add this specific ABI. What do you think?

Reply:
My concern is the naming - "firmware update policy".
On one hand, NO_FW_UPDATE is already reported in TDI report. Not sure why we duplicate here?
On the other hand, it is unclear `MEAS_FRESH_CAP` is part of firmware update policy. To me, it is more like device capability/configuration. Host cannot set it.

If the purpose to add return more configuration/attribute, I suggest we use a different name, such as "SPDM configuration" and "TDISP configuration".

Reply:
I renamed that section to `TDISP and SPDM configuration`, and replaced the `sbi_covg_get_device_update_attrs()`ABI with a `sbi_covg_get_device_spdm_attrs()` one. Let me know if that looks better to you now.